tools/webpack.config.ts

```diff
@@ -1,9 +1,27 @@
 import { resolve } from 'path';
 import * as BundleTracker from 'webpack-bundle-tracker';
 import * as webpack from 'webpack';
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const assets = require('./webpack.assets.json');
 
+// Currently we're using style-loader for local dev
+// because MiniCssExtractPlugin doesn't support
+// HMR as yet. When this changes we can switch
```

Comment:
Is there an issue in MiniCssExtractPlugin for this feature that we can link to?

Reply:
Added link to issue.